projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-add-widget-endpoint.php

```diff
@@ -5,6 +5,49 @@
  * Endpoint: https://public-api.wordpress.com/rest/v1.1/sites/$site/widgets/new
  */
 
+new WPCOM_JSON_API_Add_Widgets_Endpoint(
+	array(
+		'description'          => 'Activate a group of widgets on a site. The bulk version of using the /new endpoint',
+		'group'                => '__do_not_document',
+		'stat'                 => 'widgets:new:bulk',
+		'force'                => 'wpcom',
```

Comment:
Heh, I think this line here (in the wpcom copy) makes it impossible to hit this endpoint via public-api.wordpress.com for testing. Oh well.